### DISCUSSION TITLE
External icon path for treeview item of vscode plugin

### DISCUSSION THREAD

**Discussion by jimwu11235** [eclipse-theia/theia#11852] [Q&A]
I want to assign icon path which is outside of plugin root folder, but the console shows following error:
![image](https://user-images.githubusercontent.com/10918640/200733017-257531ae-9ed8-4e18-978e-7a4072f7eb20.png)

This usage works fine in vscode.
Is there some modification I can perform to use external icon path, or it is just the limit of theia ?

**Comment by msujew** [comment]
Hey @jimwu11235,

I don't believe that we have implemented something in particular which prevents us from reading those files, so it might be a permissions issue on your side. So it shouldn't be a Theia issue, at least in theory.

Note that I wouldn't recommend storing extension data outside of the extension itself. Is there a special reason for you doing this? It seems very hacky and prone to errors.

**Reply by jimwu11235** [reply]
Hi @msujew thansks for replying

I am trying to show icons provided by another extension. The error message indicates there is some root errors in express.
I am using Theia electron package to launch my extension.
Since this extension can work in vscode platform, is there some tips to prevent this permission issue under Theia platform ?

**Reply by msujew** [reply]
No idea, never encountered such an issue. It'd nice if you could provide a reproducible example so the team could test this and potentially provide a fix for this if it indeed a Theia issue.
